docs/README.md

```diff
@@ -77,3 +169,100 @@ litexa test
 # deploy to aws
 litexa deploy
 ```
+</div>
+</div>
+
+
+
+
+<div class="faquestions">
+
+<FAQuestion>
+
+## How much does Litexa cost? Where does it come from?
+
+Litexa is *free!*
```

Comment:
Since that came up in a recent GitHub issue, perhaps good to mention AWS cost while you're already updating this? Unless that's planned for a future update.

Reply:
Future update. I think we want to be clearer about exactly how litexa uses AWS resources.